performance/client/src/__tests__/primary/ExplorerAPIs/get-unixfiles-content.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright IBM Corporation 2020
+ */
+
+import WrkTestCase from "../../../testcase/wrk";
+import { getBasicAuthorizationHeader } from "../../../utils";
+
+class ExplorerApiUnixFileContentTest extends WrkTestCase {
+  fetchZoweVersions = true;
+
+  name = "Test explorer data sets api endpoint /unixfiles/{path}";
+  endpoint = '/api/v2/unixfiles/u/user2/zoweTestFiles/testFile';
```

Comment:
Similar to data set test, this will fail on Marist. We can find something in /etc folder which exist on most of the systems.

Reply:
I modified this test to instead pull the content of `/usr/lpp/tcpip/man/C/cat1/host.1`, which should (as far as we can tell) exist on most systems.